docs/dashboards/dashboards-with-vue.md

```diff
@@ -1,69 +1,75 @@
+```typescript
+import Highcharts from 'highcharts';
+import Dashboards from '@highcharts/dashboards';
+import DataGrid from '@highcharts/dashboards/datagrid';
 
-    Dashboards.PluginHandler.addPlugin(Dashboards.HighchartsPlugin);
-    Dashboards.PluginHandler.addPlugin(Dashboards.DataGridPlugin);
-    ```
+Dashboards.HighchartsPlugin.custom.connectHighcharts(Highcharts);
+Dashboards.DataGridPlugin.custom.connectDataGrid(DataGrid);
 
-    __Please Note:__ If you are using the Visual Studio Code editor with the
-    Volar extension, you should change the extension setting
-    `"vue.server.maxFileSize"` to a value of at least `25000000` bytes to get
-    full editor support for all Highcharts modules. You will find it in the menu
-    `File` -> `Preferences` -> `Settings`, where you have to scroll to the
-    `Vue: Max File Size` value.
+Dashboards.PluginHandler.addPlugin(Dashboards.HighchartsPlugin);
+Dashboards.PluginHandler.addPlugin(Dashboards.DataGridPlugin);
+```
 
-4. Create an HTML structure for the dashboard.  
-    There are two ways to do it:
-    #### Use the Dashboards layout system.  
-    To do that, first import the `layout` module and initialize it:
-    ```typescript
-    import LayoutModule from '@highcharts/dashboards/modules/layout';
+__Please Note:__ If you are using the **Visual Studio Code** editor with the
+**Volar** extension, you should change the extension setting
+`"vue.server.maxFileSize"` to a value of at least `25000000` bytes to get
+full editor support for all Highcharts modules. You will find it in the menu
+`File` -> `Preferences` -> `Settings`, where you have to scroll to the
+`Vue: Max File Size` value.
 
-    LayoutModule(Dashboards);
-    ```
-    Then add a div where you want to render the dashboard:
-    ```html
-    <div ref="dashboardContainer"></div>
-    ```
-    #### Declare your HTML structure.
-    Read more in the [documentation](https://www.highcharts.com/docs/dashboards/layout-description).
+## 4. Create an HTML structure for the dashboard
 
-5. Create a dashboard using the factory function `Dashboards.board`. The function takes three arguments:
-    - `container` - the element where the dashboard will be rendered, can be an id of the element or the direct reference to the element
-    - `options` - the options object for the dashboard
-    - `isAsync` - whether the dashboard should be rendered asynchronously or not - functional when using external data resources
+There are two ways to do it:
+
+### Use the Dashboards layout system
+To do that, first import the `layout` module and initialize it:
+
+```typescript
+import LayoutModule from '@highcharts/dashboards/modules/layout';
+
+LayoutModule(Dashboards);
+```
+Then add a div where you want to render the dashboard:
+```html
+<div ref="dashboardContainer"></div>
+```
+### Declare your HTML structure.
+Read more in the [documentation](https://www.highcharts.com/docs/dashboards/layout-description).
+
+## 5. Create a dashboard
```

Comment:
Is this number needed here?

Reply:
It's the last step in the process, so I believe yes